test/code/providers/Container_ImageInventory_Class_Provider_UnitTest.cpp

```diff
@@ -93,30 +105,62 @@ class ImageInventoryTest : public CppUnit::TestFixture
 		StandardTestEnumerateInstances<mi::Container_ImageInventory_Class_Provider>(m_keyNames, context, CALL_LOCATION(errMsg));
 
 		// Get images using command line
-		CPPUNIT_ASSERT(!system("docker images -q --no-trunc > /tmp/docker_image_ids.txt"));
+		CPPUNIT_ASSERT(!system("docker images --no-trunc > /tmp/docker_image_ids.txt"));
```

Comment:
Blech. Is this what all the test code is doing? Nothing is using SCXProcess::Run() from the PAL, which gives you much greater control about looking at stdout/stderr?

I noted that providers/Container_DaemonEvent_Class_Provider_UnitTest.cpp is using SCXProcess::Run. Please do that here, it lets you look at stderr and make sure that nothing came in on that channel.